control/grpc.py

```diff
@@ -132,57 +193,65 @@ def delete_bdev(self, request, context=None):
 
         self.logger.info(f"Received request to delete bdev {request.bdev_name}")
         use_excep = None
```

Comment:
@gbregman 
Could we streamline `use_excep` logic by raising the exception when it occurs?

Reply:
@baum as I said before I prefer not to nest such logic too much and just a flag and do the logic elsewhere. Especially when dealing with exceptions. The way it's done here we get into the existing exception handling and return the proper return code. Raising the exception instead of setting the flag will require duplicating this handling code.

Reply:
@gbregman see the suggested [patch](https://github.com/baum/ceph-nvmeof/commit/07556979f332f405fa4adf2e116e49d479cbd17f), Let me know how you feel about it.

Reply:
@baum this is not good. As I said before I want the existing exception handler to get activated:

            except Exception as ex:
                self.logger.error(f"delete_bdev failed with: \n {ex}")
                if context:
                    context.set_code(grpc.StatusCode.INTERNAL)
                    context.set_details(f"{ex}")
                return pb2.req_status()